specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/machineLearningServices.json

```diff
@@ -3155,6 +3208,9 @@
                   "description": "The last operation on ComputeInstance.",
                   "$ref": "#/definitions/ComputeInstanceLastOperation",
                   "readOnly": true
+                },
+                "computeSchedules": {
+                  "$ref": "#/definitions/ComputeSchedules"
```

Comment:
1. add description
2. new schedules property should be readOnly

Reply:
Updated description. Not making it readonly as it is user input for create CI api.